server/trace/mock_trace_test.go

```diff
@@ -129,6 +162,63 @@ func TestMockTraceServer_BatchWriteSpans(t *testing.T) {
 	}
 }
 
+func TestMockTraceServer_BatchWriteSpans_Error(t *testing.T) {
```

Comment:
Maybe split this up for testing invalid timestamps and missing fields, it's a little packed.

Reply:
done